conf/test/orientdb/grakn.properties

```diff
@@ -48,12 +47,12 @@ blockingLoader.batch-size=1
 
 backgroundTasks.post-processing-delay=200000
 backgroundTasks.time-lapse=20000
-tasks.kafka.bootstrap-servers=localhost:9092
-tasks.kafka.consumer.session-timeout=30000
-tasks.kafka.producer.retries=0
-tasks.kafka.producer.batch-size=16384
-tasks.kafka.producer.linger-ms=1
-tasks.kafka.producer.buffer-mem=33554432
+bootstrap.servers=localhost:9092
+consumer.session-timeout=30000
+producer.retries=0
+producer.batch-size=16384
+producer.linger-ms=1
+producer.buffer-mem=33554432
```

Comment:
These are the testing config files- I think it would be a good idea to change the few properties we modify for testing programmatically instead.